src-control/Control/RefCount.hs

```diff
@@ -200,9 +200,9 @@ class RefCounted obj where
   getRefCounter       :: obj -> RefCounter (FinaliserM obj)
 
 #ifdef NO_IGNORE_ASSERTS
-#define HAS_CALL_STACK => HasCallStack
+#define HasCallStackIfDebug HasCallStack
 #else
-#define HAS_CALL_STACK
+#define HasCallStackIfDebug ()
 #endif
```

Comment:
I prefer the all-caps macros because it makes it clear that it is not actual haskell syntax

Reply:
Do you feel that with the all-caps name, this is an improvement?

Reply:
There is, of course, the risk that this is highly dependent on what is arguably a GHC bug.